spec.md

```diff
@@ -546,6 +548,10 @@ The tags MUST be in lexical order.
 
 When using the `last` query parameter, the `n` parameter is OPTIONAL.
 
+The registry SHOULD support at least `n=1000` tags in a single response.
+The client SHOULD avoid requesting more than `n=1000` tags in a single request.
```

Comment:
yet a very large number would be a means to identify what the limit is...

```suggestion
```

Reply:
A request without any limit would return the OCI-Tag-Limit header on an error when the limit is reached, making it a more efficient way to discover the limit on an as-needed basis (the registry wouldn't fail when the returned number of tags is less than its limit).

Reply:
That sounds like another/different client SHOULD recommendation.

Reply:
It's a MUST requirement on the registry that comes out of the next line: "If the registry is unable to return all tags requested by the client, the registry MUST return a `413 Content Too Large`..."

Reply:
I removed the suggested limit from the PR.